### PR TITLE
Small rescale for DT DOC3 shifts

### PR DESCRIPTION
**Pull request by jfernan2**

### CODE REVIEW

File: dqmgui/style/DTRenderPlugin.cc [hunk 1]
```diff
@@ -1426,7 +1426,7 @@ class DTRenderPlugin : public DQMRenderPlugin
       if(o.name.find("TTSSummary") != std::string::npos)
       {
         TH2F * histo =  dynamic_cast<TH2F*>( o.object );
-        int nBinsX = histo->GetNbinsX();
+        int nBinsX = histo->GetN
```
Comment (threus) [inline dqmgui/style/DTRenderPlugin.cc]: @jfernan2 please fix the typo and test it in the local GUI.

**Comment by jfernan2** [comment]
Sorry. 
Done. Tested on:
http://fanae43.geol.uniovi.es:8080/dqm/offline/start?runnr=296116;dataset=/StreamExpress/Run2017A-Express-v1/DQMIO;sampletype=offline_data;filter=all;referencepos=overlay;referenceshow=customise;referencenorm=True;referenceobj1=refobj;referenceobj2=none;referenceobj3=none;referenceobj4=none;search=;striptype=object;stripruns=;stripaxis=run;stripomit=none;workspace=Shift;size=M;root=00%20Shift/DT;focus=;zoom=no;
Thanks

**Comment by threus** [comment]
@h4d4 , Tests are OK. Could you please merge this?
Thanks,
Tomas for DQM

**Comment by jfernan2** [comment]
Thank you!

**Pull request merged**